build.gradle

```diff
@@ -51,8 +51,7 @@ dependencies {
 
     api 'joda-time:joda-time:' + jodaVersion, optional
     api 'com.github.jknack:handlebars-helpers:' + handlebarsVersion, optional
-    api 'com.github.jknack:handlebars-humanize:' + handlebarsVersion, optional
-    api 'com.github.jknack:handlebars-jackson2:' + handlebarsVersion, optional
```

Comment:
Note: renamed to `handlebars-jackson` in https://github.com/jknack/handlebars.java/commit/d8ff863d36aa0a27dcecce7bdc194f6d45a34676